pkg/codegen/internal/test/testdata/simple-enum-schema/python/pulumi_plant/tree/v1/rubber_tree.py

```diff
@@ -88,6 +88,28 @@ def size(self, value: Optional[pulumi.Input['TreeSize']]):
         pulumi.set(self, "size", value)
 
 
+@pulumi.input_type
+class _RubberTreeState:
+    def __init__(__self__, *,
+                 farm: Optional[pulumi.Input[Union['Farm', str]]] = None):
+        """
+        Input properties used for looking up and filtering RubberTree resources.
+        """
+        if farm is None:
+            farm = '(unknown)'
+        if farm is not None:
+            pulumi.set(__self__, "farm", farm)
```

Comment:
Very nitty but the inconsistent use of single and double quotes is a personal pet peeve. We seem to generally opt for double quotes in our python code, but I'm guessing because of the generated code and the burden of having to escape the single quotes is why we've ended up in this situation. An alternative we can use in code generation is to use `%q` instead of `\"%s\"` or `'%s'`. [[Docs]](https://golang.org/pkg/fmt/#hdr-Printing:~:text=%25q%09a%20double%2Dquoted%20string%20safely%20escaped%20with%20Go%20syntax)

I've noted some spots in gen.go below where we could utilize this.

Reply:
It's a pet peeve of mine as well! But, I've just shrugged it off in this generated code since we're already inconsistent throughout. In any case, I fixed all the lines that I modified in this PR to use double quotes. We can follow-up later as we have time to improve the rest of the generated code so it's consistent. My initial thought on a style guideline would be to use double quotes everywhere, but perhaps stick to single quotes for forward reference type hints?